package/crds/kms.aws.crossplane.io_aliases.yaml

```diff
@@ -84,9 +75,9 @@ spec:
                       For example: \n    * Key ID: 1234abcd-12ab-34cd-56ef-1234567890ab
                       \n    * Key ARN: arn:aws:kms:us-east-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab
                       \n To get the key ID and key ARN for a CMK, use ListKeys or
-                      DescribeKey.'
+                      DescribeKey."
                     type: string
-                  targetKeyIDRef:
+                  targetKeyIdRef:
```

Comment:
Is it possible to maintain the old casing here? I believe if we did this would not be a breaking change.

Reply:
Oh nevermind - just realised this is a new managed resource so it's not a breaking change anyhow.